src/nvim/message.c

```diff
@@ -743,7 +743,7 @@ void ex_messages(exarg_T *eap)
   msg_hist_off = TRUE;
 
   s = os_getenv("LANG");
-  if (s != NULL && *s != NUL)
+  if (!s)
```

Comment:
The "Message Maintainer" message will be removed soon anyways (what the hell is a messages maintainer?). I didn't realize this was for that :)